synapse/models/geospace.py

```diff
@@ -374,6 +382,19 @@ def getModelDefs(self):
 
                     )),
 
+                    ('geo:telem', {}, (
+                        ('time', ('time', {}), {
+                            'doc': 'The time that the node was at the position.'}),
+                        ('desc', ('str', {}), {
+                            'doc': 'A description of the telemetry sample.'}),
+                        ('latlong', ('geo:latlong', {}), {
+                            'doc': 'The latitude/longitude reading at the time.'}),
+                        ('place', ('geo:place', {}), {
+                            'doc': 'The place which includes the latlong value.'}),
+                        ('place:name', ('geo:name', {}), {
```

Comment:
Is :place:name on the geo:telem node duplicative? This property already exists on the geo:place node referenced by :place. Currently, other "telem" model elements record :latlong and :place, but not :place:name.

Reply:
This was purposely modeled to capture polling places with names that don't correspond to the overall place. "3rd district polling place" != "foo county library" etc

Reply:
:+1: